web/turkey/survey/static/survey/js/auditors/scrolled_vertical_pixels_total.js

```diff
@@ -0,0 +1,34 @@
+var AuditorScrolledVerticalPixelsTotal = {
```

Comment:
Is there a good reason not to combine the horizontal and vertical statistics collection?